_posts/2025-02-28-esrf-trip-report.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: ESRF Trip Report
+subtitle: Mix and Inject Serial Crystallography at ESRF
+author: Kevin Dalton
+usemathjax: true
```

Comment:
Just noting for posterity that the `usemathjax` line is only necessary when the post includes latex. There isn't *really* any downside in always including it, other than a) making the page load ever-so-slightly slower, and b) it bothering my pedantic sensibilities.

Reply:
awesome, thanks!